code/modules/mob/living/bot/cleanbot.dm

```diff
@@ -75,7 +75,6 @@ var/list/cleanbot_types // Going to use this to generate a list of types once th
 	if(!path.len)
 		path = AStar(loc, target.loc, /turf/proc/CardinalTurfsWithAccess, /turf/proc/Distance, 0, 30, id = botcard)
 		if(!path)
-			custom_emote(2, "can't reach \the [target.name] and is giving up for now.")
```

Comment:
Theres quite a few unnecessary emotes in there. Please also remove the one about "Cleanbot starts cleaning the dirt"

I'd say remove that debug log too, its not really relevant in life play. Comment it out so we can re-enable it later when someone is working on them

The damn things are too chatty